src/components/audio-player-view/audio-player-view.scss

```diff
@@ -68,11 +68,9 @@ $ma-player-border-radius: var(--ma-player-border-radius);
       left: 0;
       height: 100%;
       width: 100%;
-      filter: blur(48px) brightness(70%);
-      -webkit-filter: blur(48px) brightness(70%);
-      background-position: center;
-      background-repeat: no-repeat;
-      background-size: cover;
+      filter: blur(24px);
+      opacity: 0.5;
+      background: lightgray 50% / cover no-repeat;
```

Comment:
why remove the brightness ?

Reply:
the style was not aligned with the requirements on figme, i took the style as it is from the figma on dev mode

Reply:
not got you, that’s exactly what I’m saying that it was not according to the spec and that’s why i fixed it to be according to the spec (and after talking with dana)

Reply:
ok